model/hetzner_host.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require_relative "../model"
+
+class HetznerHost < Sequel::Model
+  one_to_one :vm_host, key: :id
+
+  PROVIDER_NAME = "hetzner".freeze
```

Comment:
String literals are automatically frozen given the `# frozen_string_literal: true` written into the top of the file
```suggestion
  PROVIDER_NAME = "hetzner"
```